### PR TITLE
Custom html title

### PR DESCRIPTION
**Pull request by mathcarignani**
issue #107 

Any feedback are welcome, it's my first PR ;)

### CODE REVIEW

File: lib/snapshot/snapshot_config.rb [hunk 1]
```diff
@@ -105,6 +107,10 @@ def set_defaults
         setup_for_language_change: empty,
         teardown_language: empty
       }
+
+      name = self.project_name
+      self.html_title = name ? name : 'KrauseFx/snapshot'
```
Comment (KrauseFx) [inline lib/snapshot/snapshot_config.rb]: You can replace this to
```ruby
self.html_title = self.project_name || 'KrauseFx/snapshot'
```

File: README.md [hunk 2]
```diff
@@ -252,6 +252,12 @@ I'll try to keep the script up to date. If you need to change the iOS version, y
 ios_version "9.0"
 ```
 
+### Html Title
+By default, ```snapshot``` will try to set the html title with the project name. If you want to specify a title, you can do it like this:
```
Comment (KrauseFx) [inline README.md]: Please change to
```
If you want to change the title of the generated HTML page, you can use:
```

**Comment by KrauseFx** [comment]
Thanks for working on this. Looks good, I added a few comments :+1:

File: README.md (hunk 2 above)
Comment (mathcarignani) [inline README.md]: :+1: My english is not the best :(

**Comment by KrauseFx** [comment]
Thank you for your contribution @mathcarignani. This will be in the next version :+1: :rocket:

**Comment by mathcarignani** [comment]
Your work is awesome and contribute with it is a pleasure for me! :+1:

**Comment by powtac** [comment]
Top! Thanks a lot!

**Pull request merged**